packages/coinjoin/src/backend/CoinjoinMempoolController.ts

```diff
@@ -20,18 +19,16 @@ export class CoinjoinMempoolController implements MempoolController {
 
     async update() {
         const txids = await this.client.fetchMempoolTxids();
-        const entries = await Promise.allSettled(
-            txids.map(async txid => {
-                const tx =
-                    this.mempool[txid] ??
-                    (await this.client.fetchTransaction(txid, {
-                        identity: this.client.getIdentityForBlock(undefined),
-                    }));
-                return [txid, tx] as const;
-            }),
+        const entries = await promiseAllSequence(
+            txids.map(
+                txid => () =>
+                    this.mempool[txid]
+                        ? Promise.resolve(this.mempool[txid])
+                        : this.client.fetchTransaction(txid).catch(() => undefined),
```

Comment:
So we do not care about fetchTransaction failing here regardless of the reason it fails?

Reply:
Yes. It's still tried three-times, but if it fails, it fails. Mempool syncing has to be reworked soon nevertheless.